src/features/lessons/list-of-lessons/list-of-lessons.js

```diff
@@ -120,15 +124,23 @@ export const ListOfLessons = () => {
               onChange={handleSearchDate}
             />
           </div>
-          <div className="col-md-6 offset-md-1 text-center pl-4">
+          <div className="col-4 offset-md-1 text-center pl-4 mr-5">
             <Search onSearch={handleSearchTheme} placeholder="Search lesson" />
           </div>
-          {currentUser.role !== 3 && 
-            <div className="col-md-2 offset-md-1 text-right">
-              <Button onClick={addLesson} variant="warning">
-                <Icon icon="Plus" className="icon" />
-                  Add a Lesson
-              </Button>
+          {currentUser.role !== 3 &&
+            <div className="col-4 offset-md-1 text-right align-items-center ml-5 pl-5">
+              <div className="btn-group" >
+                <Button onClick={downloadThemes} type="button" className="btn btn-warning">
```

Comment:
Maybe better write onClick like this: `onClick={()=>(history.push(paths.THEMES_DOWNLOAD)}`